APMrover2/mode.h

```diff
@@ -153,6 +149,7 @@ class Mode
     Location _destination;      // destination Location when in Guided_WP
     float _distance_to_destination; // distance from vehicle to final destination in meters
     bool _reached_destination;  // true once the vehicle has reached the destination
+    float _desired_lat_accel;   // desired lateral acceleration in m/s/s
```

Comment:
Do we even need to store this now if the controller is doing it for us?  Can it just be a local in the various `update` functions?

Reply:
haha!  yes, I think the same thing.  we can actually just remove it now but I figured I'd do that in a future change.